### DISCUSSION TITLE
Telegram Api to use?

### DISCUSSION THREAD

**Discussion by Aqshola** [mgilangjanuar/teledrive#4] [Q&A] [answered]
I've read telegram api docs, look like they didnt have rest api documentation, so did we will built everything from scratch? Or using library like [Airgram](https://github.com/airgram/airgram)

**Comment by mgilangjanuar** [comment, marked as answer]
Yes, the Telegram API using MTProto. Actually I just install the [gramjs](https://gram.js.org/) on the `server`. But, Airgram looks more interesting *(especially from the number of the stars 🤣)*

Thank you for the suggestion anw!
